src/components/Landing/QAs.tsx

```diff
@@ -89,38 +116,58 @@ export default function QAs() {
     {
       q: t`What's going on under the hood?`,
       a: [
-        t`This website (juicebox.money) connects to the Juicebox protocol's smart contracts, deployed on the Ethereum network. (note: anyone else can make a website that also connects to these same smart contracts. For now, don't trust any site other than this one to access the Juicebox protocol.)`,
+        t`This website (juicebox.money) connects to the Juicebox protocol's smart contracts, deployed on the Ethereum network. (note: anyone else can make a website that also connects to these same smart contracts. For now, don't trust any site other than this one to access the Juicebox protocol).`,
         t`Creating a Juicebox project mints you an NFT (ERC-721) representing ownership over it. Whoever owns this NFT can configure the rules of the game and how payouts are distributed.`,
-        t`The project's tokens that are minted and distributed as a result of a received payment are ERC-20's. The distribution schedule is proportional to payments recieved, weighted by the project's discount rate over time.`,
+        t`The project's tokens that are minted and distributed as a result of a received payment are ERC-20's. The amount of tokens minted and distributed are proportional to the volume of payments recieved, weighted by the project's discount rate over time.`,
       ],
     },
     {
       q: t`How decentralized is Juicebox?`,
       a: [
         t`Juicebox is a governance-minimal protocol. There are only a few levers that can be tuned, none of which impose changes for users without their consent. The Juicebox governance smart contract can adjust these levers.`,
-        t`At the start, power over the governance smart contract is held by Juicebox's founding contributors. The intent is to soon transfer the power to a community of token holders.`,
+        t`The Juicebox protocol is governed by a community of $JBX token holders who vote on proposals fortnightly.`,
```

Comment:
thoughts on moving from $JBX to JBX

Reply:
Yeah for sure, should keep consistent in any case and everywhere else has ditched the $ already